packages/terra-graphs/tests/wdio/Graphs/BarGraph/BasicBarGraphs/GroupedBars-spec.js

```diff
@@ -0,0 +1,12 @@
+Terra.describeViewports('Grouped bars', ['tiny', 'medium', 'large'], () => {
+  describe('Validates', () => {
```

Comment:
We should use a more meaningful title that describes this test. Same suggestion for all the other tests.

```suggestion
  describe('GroupedBars', () => {
```